event_spec.md

```diff
@@ -232,14 +228,37 @@ The following attributes are REQUIRED to be present in all Cloud Native Events:
     [RFC 3339](https://tools.ietf.org/html/rfc3339)
 
 
+### data
+
+This is defined in the next section.
+
 ## Event Data
 
-As defined by the term [Data](#data), Cloud Native Events data MAY include domain-specific
+Cloud Native Events data includes a version number and a generic `Values` field or a Redfish hardware specific `Data`field.
```

Comment:
add PTP specific `Data` Field

Reply:
My understanding is for now PTP fits into the more generic event.Data structure, while HW events use a specific hwevent.Data as described below. The `Data` field in L#237 refers to `hwevent.Data.Data`.
```
event.Data
type Data struct {
	Version string      `json:"version" example:"v1"`
	Values  []DataValue `json:"values"`
}

hwevent.Data
type Data struct {
	Version string        `json:"version" example:"v1"`
	Data    *RedfishEvent `json:"data"`
}
```

Reply:
so its data inside data . So what we need to do is cut out data section make make different schema to represent various data schema .

Reply:
You should call it value since RedfishEvent type is a value of data type .

Reply:
I plan to change this in the future. See proposed change for issue https://github.com/redhat-cne/sdk-go/issues/24